trk/src/Screens/ClimbCreate/index.js

```diff
@@ -303,6 +310,7 @@ const ClimbInputData = (props) => {
                 placeholderTextColor={"#b1b1b3"}
                 onChangeText={setInfo}
                 placeholder="Enter more info"
+                multiline={true}
```

Comment:
this is to fix the fact that the Climb info input text did not wrap onto the next line